src/datasources/userDataCreateUser.datasource.js

```diff
@@ -0,0 +1,18 @@
+import config from "../config";
+import request from "superagent";
```

Comment:
vi skal ikke bruge suger agent - brug context.fetcher - der er noget generisk kode i som skal eksekvereres for alle request

Reply:
Done